tests/example/input_output/input_output.dl

```diff
@@ -0,0 +1,172 @@
+// Souffle - A Datalog Compiler
+// Copyright (c) 2018, The Souffle Developers. All rights reserved
+// Licensed under the Universal Permissive License v 1.0 as shown at:
+// - https://opensource.org/licenses/UPL
+// - <souffle root>/licenses/SOUFFLE-UPL.txt
+
+// input_output.dl
```

Comment:
A brief description of what this test is for is needed.

Reply:
Will do.